content/pages/meetup/2024/1002-nycu.rst

```diff
@@ -0,0 +1,141 @@
+========================================
+Meetup 2024 September 25th in NYCU
```

Comment:
This title needs to be updated to **October 2nd**.

Reply:
FIx.